config/admin-functions/lockdown.sh

```diff
@@ -30,7 +30,7 @@ mount /dev/sda /mnt || mount /dev/sda1 /mnt || (echo "Secure boot keys not found
 sbsign --key=/mnt/DB.key --cert=/mnt/DB.crt --output /boot/efi/EFI/debian/VxLinux-signed.efi /tmp/linux.efi
 
 # Now install it 
-bash setup-boot-entry.sh
+bash /vx/admin/admin-functions/setup-boot-entry.sh
```

Comment:
I noticed that a lot of the admin-functions scripts pull out these paths into variables. E.g. at the top of `show-vx-suite-admin-menu.sh`:

```bash
: "${VX_FUNCTIONS_ROOT:="$(dirname "$0")"}"
: "${VX_CONFIG_ROOT:="/vx/config"}"
```

Do you think something like that would be appropriate here?

Reply:
Yes, +1, we should do the `VX_FUNCTIONS_ROOT` thing that defaults to `dirname "$0"`